docs/features/latency.md

```diff
@@ -0,0 +1,231 @@
+it will be stated that TS is the true difference between the connection
+start time and the time when the sending time has been declared when
+the sender application is calling any of the `srt_send*` functions
+(see [`srt_sendmsg2`](../API/API-functions.md#srt_sendmsg2) for details).
+
+
+SRT latency components
+======================
+
+To understand the latency components we need also other definitions:
+
+* ETS: expected arrival time. This is the time of the packet when it
+"should" arrive according to its TS
+
+* PTS: packet's play time. It's the time when SRT gives up the packet
+to the `srt_recvmsg2` call (that is, it sets up the IN flag in epoll
+and resumes the blocked function call, if it was in blocking mode).
+
+* STS: the time declared as the sending time when the packet was
+scheduled for sending at the sender side (if you don't use the
+declared time, by default it's the monotonic time taken when this
+function was called), which is represented by TS.
+
+* RTS: the same as STS, but calculated at the receiver side. The
+only way to extract it is by using some initial statements.
+
+The "true latency" for a particular packet in SRT can be simply defined as:
+
+* `TD = PTS - STS`
+
+Note that this is a stable definition (independent on the packet),
+but this value is not really controllable. So let's define the PTS
+for the packet `x`:
+
+* `PTS[x] = ETS[x] + LATENCY + DRIFT`
+
+where `LATENCY` is the negotiated latency value (out of the
+`SRTO_RCVLATENCY` on the agent and `SRTO_PEERLATENCY` on the peer)
+and DRIFT will be described later (for simplification you can
+state it's initially 0).
+
+These components undergo the following formula:
+
+* `ETS[x] = start_time + TS[x]`
+
+Note that it's not possible to simply define it basing on STS
+because sender and receiver are two different machines that can only
+see one another through the network, but their clocks are separate,
+and can even run on different or changing speeds, while the only
+visible phenomena happen only at a packet arrival machine. This
+above formula, however, allows us to define the start time because
+we state the following for the very first data packet:
+
+* `ETS[0] = ATS[0]`
+
+This means that from this formula we can define the start time:
+
+* `start_time = ATS[0] - TS[0]`
+
+Therefore we can state that if we have two identical clocks on
+both machines with identical time bases and speeds, then:
+
+* `ATS[x] = program_delay[x] + network_delay[x] + STS[x]`
+
+(The only problem with treating this above formula too seriously
+is that there doesn't exist the common clock base for two
+network-communicating machines, so these components should be
+treated as something that does exist, but isn't exactly measurable).
+
+But even if there is still this formula for ATS, it doesn't
+apply to the real latency - this one is based strictly on ETS.
+But you can apply this formula for the very first arriving
+packet, because for this one they are equal: `ATS[0] = ETS[0]`.
```

Comment:
```suggestion
This formula for ATS doesn't apply to the real latency, which is based strictly 
on ETS. But you can apply this formula for the very first arriving packet, 
because in this case they are equal: `ATS[0] = ETS[0]`.
```